src/Validator.php

```diff
@@ -24,16 +22,76 @@ public static function validateClassName(string $className, string $errorMessage
         if (!preg_match('/^[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*$/', $className)) {
             $errorMessage = $errorMessage ?: sprintf('"%s" is not valid as a PHP class name (it must start with a letter or underscore, followed by any number of letters, numbers, or underscores)', $className);
 
-            throw new RuntimeCommandException($errorMessage);
+            throw new \InvalidArgumentException($errorMessage);
```

Comment:
These changes come back to the "ugly" exception message, what about to use `Symfony\Component\Console\Exception\InvalidArgumentException` instead? or create your own `InvalidArgumentException` that extends from it?

Reply:
Good catch. It doesn't make sense MOST of the time (i.e. it doesn't make sense if these are callbacks to a Question validation, where the exception is caught), but in case these methods are ever called in a different situation, better to throw the "prettier" exception.